doc/ref/chap-type-method.md

```diff
@@ -27,6 +27,25 @@ This chapter in the [Oils Reference](index.html) describes YSH types and methods
 
 ### replace()
 
+Replace substrings or eggex matches within a string.
+
+    var mystr = "code is 1234"
+    = mystr=>replace("code", "Code")  # => "Code is 1234"
```

Comment:
FWIW I started writing this `mystr => replace(...)` with spaces

vs. `call mylist->pop()`

That is up for debate, but I think it makes them look more different and is a nice style